lib/compress.js

```diff
@@ -1769,8 +1769,52 @@ merge(Compressor.prototype, {
                 "substr",
                 "substring",
                 "trim",
-            ].concat(object_fns)),
+            ].concat(object_fns),
         };
+        var static_fns = {
+            Array: [
+                "isArray",
+            ],
+            Math: [
+                "abs",
+                "acos",
+                "asin",
+                "atan",
+                "ceil",
+                "cos",
+                "exp",
+                "floor",
+                "log",
+                "round",
+                "sin",
+                "sqrt",
+                "tan",
+                "atan2",
+                "pow",
+                "max",
+                "min"
+            ],
+            Number: [
+                "isFinite",
+                "isNaN",
+            ],
+            Object: [
+                "keys",
+                "create",
+                "getPrototypeOf",
```

Comment:
Actually, it does throw exceptions on a bad day:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/getPrototypeOf
> In ES5, it will throw a TypeError exception if the obj parameter isn't an object. In ES2015, the parameter will be coerced to an Object.

Removed in https://github.com/mishoo/UglifyJS2/pull/2211/commits/f46f0743c7cf8cdfc15a2471b647aacd080cbbc3